_posts/2019-07-30-Rates-Rebate-Beta-Research-Report.md

```diff
@@ -0,0 +1,468 @@
+
+### 2. Conversations with council staff & Department Internal Affairs
+
+Conversations were conducted with Tauranga City Council, Hutt City Council, and Kāpiti District Council throughout the Beta phase. We observed the processing of applications at each council, and the unique system each council used. Additional conversations were held with Wairarapa councils to understand their process and pain points.
+
+We worked alongside subject matter experts from Rates Rebates team, within the Service Delivery and Operations branch at DIA.
+
+### 3. Online surveys
+
+Survey research targeted existing visitors to the Rates Rebates section of govt.nz. Two surveys ran during the Beta project. The initial survey ran for two weeks at the start of the project, before any changes had been made to the content. A follow up survey ran for three weeks during the Beta trial to gauge visitors perception of the process and the changes implemented on the online application form.
+
+### 4. Analytics analysis
+
+We identified a set of data that we could pull from analytics on the original Rates Rebates section of govt.nz. This helped us understand current baseline behaviour, search patterns and plan appropriate content structure. The process and the results of this are available as a [blog: ‘Rates Rebate - content changes lead to better experience, more users’](https://www.digital.govt.nz/blog/rates-rebate-content-changes-lead-to-better-experience-more-users/)
+
+## Outputs
+
+### Synthesis
+
+Once we had gathered all the data, we went through multiple steps of analysis and outputs. Because this activity was conducted in parallel with the Beta build, we were only able to implement improvements based on early insights immediately into the design of the service. Additional insights have surfaced as we have put the Beta online trial into production. It is hoped these can drive further improvements beyond Beta.
+
+### The results from this research inform how we might
+
+1. Help increase the uptake of the Rates Rebate
+2. Improve customer experience and satisfaction levels
+3. Decrease the cost of processing applications for councils
+4. Improve the ability to digitally deliver this and other services to councils
+5. Support the case to remove the statutory declaration (witnessed signature) requirement
+6. Reuse these insights for other use cases that have similar customer profiles
+
+This report was produced in parallel with the [Rates Rebates Beta Recommendations Report](https://docs.google.com/document/d/10JO2XS1g-PYD26S4P8FjSMD6Q3tJB72vnw55vuatqWQ/edit?usp=sharing) which documents the key results of the trial, presents options for next steps, and makes a set of recommendations.
```

Comment:
Can we link this to the toolkit version of the report once it is up please

Reply:
Sure, will change it :+1: